cmd/can/main.go

```diff
@@ -39,12 +44,14 @@ func main() {
 		"package": cfg.Template.BasePackageName,
 	})
 
-	_, err = tree.Traverse(apiSpec, e.BuildRenderNode())
+	_, err = tree.Traverse(apiSpec, engine.BuildRenderNode())
 	if err != nil {
 		fmt.Println(err.Error())
 		os.Exit(1)
 	}
 }
+
+// setStrategy creates and applies the renderer for the given strategy. An error is returned if the strategy is invalid
```

Comment:
This comment is valuable, because it reiterates what can be derived from the function signature in clear English. How do you feel about using this space to clarify the intent of this functionality? Unfamiliar eyes may not know what a strategy is in this context or why we use it.

Reply:
`creates and applies the renderer` is pretty clear if you ask me.. What would you suggest?